test/src/test/java/hudson/security/csrf/CrumbExclusionTest.java

```diff
@@ -69,7 +70,9 @@ public void pathInfo() throws Exception {
                     assertThat("error message using " + path, x.getResponse().getContentAsString(), containsString("No valid crumb was included in the request"));
                     break;
                 case 400: // from Jetty
-                    assertThat("error message using " + path, x.getResponse().getContentAsString(), containsString("Ambiguous path parameter"));
+                    WebResponse response = x.getResponse();
+                    String expected = response.getResponseHeaderValue("Server").contains("Jetty(9") ? "Ambiguous path parameter" : "Ambiguous URI path parameter";
+                    assertThat("error message using " + path, x.getResponse().getContentAsString(), containsString(expected));
```

Comment:
or simply
```suggestion
                    assertThat("error message using " + path, x.getResponse().getContentAsString(), containsString("path parameter"));
```
?

Reply:
I do not see a reason why we should simply do that.

Reply:
Well, because it is simpler and so easier to read, and a bit less likely to break with some future Jetty update.

Reply:
The readability argument seems subjective to me, but the fragility argument seems strong.